scripts/generate_schemas.py

```diff
@@ -277,9 +297,35 @@ def convertAvro(self, avdlFile):
             print("converting", avdlFile)
         if self.verbosity > 1:
             print("running:", " ".join(args))
-        # TODO silence the Java output unless we are in verbose mode by
-        # capturing stdout/stderr here
-        subprocess.check_call(args)
+        if self.verbosity > 1:
+            subprocess.check_call(args)
+        else:
+            devnull = open(os.devnull, 'w')
+            subprocess.check_call(args, stdout=devnull, stderr=devnull)
+            devnull.close()
+
+    def extract_post_methods(self, avdlFile):
+        methods = []
+        f = open(avdlFile)
+        lines = f.readlines()
+        url = request = response = None
+        for line in lines:
```

Comment:
This is a neat method, but I worry that it's a little brittle. What would happen if someone is not quite systematic about the comment format? An alternative method (which avoids reparsing the file) is to run a regexp against all of the classes that we have generated at the end of the process. We match something like Search[x]Request, and pull out the middle token, and use this to construct the URL. Does this make sense?